README.md

```diff
@@ -130,6 +131,42 @@ python examples/mnist_decorator.py
 
 This will create a `.csv` file with information about the energy that you used to carry out the classification task, and an estimate of the CO<sub>2</sub> that you generated, complete with comparisons to common modes of transportation to give you a better idea of the order of magnitude of your emissions.
 
+# Report your emissions: LateX template
+
+We believe that an important step towards reducing carbon emissions is the generalization of emissions reporting in papers, blog posts and publications in general. Here's an example LateX snippet you might want to use:
+
+```latex
+\usepackage{hyperref}
+
+\subsection{CO2 Emission Related to Experiments}
+
+Experiments were conducted using {cloud provider} in region {region of compute}, which has a carbon efficiency of {carbon efficiency} kgCO$_2$eq/kWh. A cumulative of {hours used} hours of computation was performed on hardware of type {hardware type} (TDP of W).
```

Comment:
Should we make a second alternate template in the case where they didn't use cloud compute and instead used personal computers?  I've got a high-powered machine with GPU that I use for most of this stuff...

Reply:
I thought about that and concluded with myself that people would be free to adapt. I agree we can definitely have a "local / private infra" template, I did not want to overload the readme. @sashavor do you have an opinion?

Reply:
Can we just put 'region' instead of 'region of compute'?